samples/unit-tests/stock-tools/gui/demo.js

```diff
@@ -61,3 +61,115 @@ QUnit.test('Stocktools GUI', function (assert) {
     );
     */
 });
+
+QUnit.test('Disabling and Enabling stock Tools buttons, when series are invisible, #14192', function (assert) {
+    var wasInitCalled = false;
+
+    // Creating test controller and adding styles to button.
+    var toolsContainer = document.createElement('div');
+    toolsContainer.className += 'tools-container';
+    var button = document.createElement('button');
+    button.className += 'dummy-button';
+    button.innerHTML = 'dummy button';
+
+    toolsContainer.appendChild(button);
+
+    document.getElementById('container').parentNode.insertBefore(toolsContainer, document.getElementById('container'));
+
+    var chart = Highcharts.stockChart('container', {
+        stockTools: {
+            gui: {
+                enabled: false // disable the built-in toolbar
+            }
+        },
+        navigation: {
+            bindings: {
+                dummyButton: {
+                    className: 'dummy-button',
+                    init: function () {
+                        wasInitCalled = true;
+                    }
+                }
+            },
+            bindingsClassName: 'tools-container'
+        }
+    });
+
+    var controller = new TestController(chart);
+    toolsContainer.style.position = 'absolute';
+    toolsContainer.style['z-index'] = 99999;
+
+    controller.click(10, 10);
+    assert.equal(wasInitCalled, false, 'Init function should not be executed, when there is no series.');
```

Comment:
Any reason why this assert is in one line? It would be more readable divided into multiple lines (like in the other tests).

Reply:
FYI, we have `max-len` at 80 in our ESLint settings, but with an exception for strings. That's why ESLint didn't complain about this. But best practice is still 80.

Reply:
@pawellysy when dividing into multiple lines, it's better to do it this way:
```js
assert.ok(
    true,
    'some text'
);
```
instead of this way:
```js
assert.ok(
    true,
    'some text');
```